.azure-pipelines/docker-sonic-slave.yml

```diff
@@ -0,0 +1,98 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "0 8 * * *"
+  displayName: Build and push sonic-slave-* images
+  branches:
+    include:
+    - master
+  always: true
+
+trigger: none
```

Comment:
same as docker-sonic-mgmt, enable pr trigger, in case there is change in sonic-slave-xx folder.